plugins/modules/vmware_guest.py

```diff
@@ -3166,7 +3165,7 @@ def main():
                 num_cpus=dict(type='int'),
                 scsi=dict(type='str', choices=['buslogic', 'lsilogic', 'lsilogicsas', 'paravirtual']),
                 secure_boot=dict(type='bool'),
-                version=dict(type='str'),
+                version=dict(type='str', default='latest'),
```

Comment:
Defining a default for ``version`` might be a bit problematic. You see, if you're vSphere admin updates the environment, the module will suddenly try to update the HW version on the VMs. And will fail if they're powered on.

Without a default and with the original code, the module just didn't care about ``version`` if not set. With a default, it will _always_ be defined. This would introduce a new behavior.